### ISSUE TITLE
Possible to hide all children of certain state?

### ISSUE THREAD

**Issue by Zach-Barnes-Volusion** [ncuillery/angular-breadcrumb#96]
I'm not sure if this exists or not, but I would like to hide all child breadcrumbs of a certain state without having to declare ncyBreadcrumb {skip: true}. Is this possible, and if not, could I request this feature?

**Comment by perpetuumstef** [comment]
Yeah, ditto.  We have a use case which is very similar where we don't want to display the breadcrumb on specific states/views, but still need it tracked.  I realize we can do this by watching for the state and simply hiding it, but the use case is quite valid nonetheless.  Hopefully this is something you could include soon.  Cheers!

**Comment by ncuillery** [comment]
It could be useful :+1:
